doc/functions/cva.rst

```diff
@@ -0,0 +1,172 @@
+``cva``
+=======
+
+`CVA (Class Variant Authority)`_ is a concept from the JavaScript world and used
+by the well-known `shadcn/ui`_.
+CVA allows you to display a component with different variants (color, size, etc.),
+to create highly reusable and customizable templates. This function work best by being combined with
+`TwigComponent`_
+
+.. note::
+
+    The ``cva`` function is part of the ``HtmlExtension`` which is not
+    installed by default. Install it first:
+
+    .. code-block:: bash
+
+        $ composer require twig/html-extra
+
+    Then, on Symfony projects, install the ``twig/extra-bundle``:
+
+This is powered by a ``cva()`` Twig
+function where you define ``base`` classes that should always be present and then different
```

Comment:
If they must be present, first argument of the function / CVA should not be nullable

Reply:
This is the classes inside the base that should always be present not the base argument

Reply:
Maybe we could avoid using "base" for two distinct things then ?